### PR TITLE
MEN-1767: Make the integration version available to the UI

### PR DESCRIPTION
**Pull request by oleorhagen**
Changelog: Title

The integration version is stored in a file, and made available to
the UI through an environment variable.

Signed-off-by: Ole Petter <ole.orhagen@cfengine.com>

### CODE REVIEW

**Comment by oleorhagen** [comment]
https://tracker.mender.io/browse/MEN-1767

File: extra/release_tool.py [hunk 1]
```diff
@@ -721,6 +721,14 @@ def generate_new_tags(state, tag_avail, final):
         if not next_tag_avail[repo.git]['already_released'] and final:
             next_tag_avail[repo.git]['already_released'] = True
 
+    # Make the integration version available to the UI through an environment variable
+    # and store
+    for repo in REPOS.values():
+        if repo.git == "integration":
+            os.environ('INTEGRATION_VERSION', next_tag_avail[repo.git]['build_tag'])
```
Comment (GregorioDiStefano) [inline extra/release_tool.py]: Any point of having this line?

Comment (oleorhagen) [reply extra/release_tool.py]: Acceptance criteria

- [x] During release the integration version is stored in a file (e.g. json)
- [x] This file or its contents is mounted in a place accessible to the UI or made available to the UI *through environment variables*

File: docker-compose.yml [hunk 2]
```diff
@@ -34,6 +34,8 @@ services:
             service: mender-base
         networks:
             - mender
+        volumes:
+          - ./extra/integration-version.json:/var/www/mender-gui/dist/integration-version.json
```
Comment (oleorhagen) [inline docker-compose.yml]: @GregorioDiStefano something like this?
Not sure where it should be mounted in the container though.

**Comment by estenberg** [comment]
Thanks! Will this works for Hosted as well, @maciejmrowiec ?

**Comment by maciejmrowiec** [comment]
sure, doe this will correlate to hosted releases

**Comment by kacf** [comment]
My proposition is: Use `git describe` somewhere in the `demo` script to pass the version of integration into gui using an environment variable. If we don't want the additional Git SHA which is added if the user has made subsequent commits, then we can use `git describe --abbrev=0`.

**Comment by oleorhagen** [comment]
fyi I stopped the jenkins build

**Pull request merged**